src/axom/inlet/tests/inlet_errors.cpp

```diff
@@ -277,7 +277,7 @@ TYPED_TEST(inlet_errors, helpful_range)
   // FIXME (JBE): This appears to be needed for NVCC specs,
   // even though this test isn't built with NVCC.  Maybe the linker
   // is doing something horrible? The selection of 1.5 is random
-  axom::fmt::format("{}", 1.5);
+  (void)axom::fmt::format("{}", 1.5);
```

Comment:
It's is odd that we need to format a value and then ignore the result.
I also don't understand how this relates to NVCC.

@bmhan12 -- would you mind checking if anything breaks if we remove this line (and comment) -- especially in our cuda/hip configs? If that doesn't wrok, we should investigate this in a follow-up PR.